[2022-12-16T22:39:35.652+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2022-12-16T22:38:47.807196+00:00 [queued]>
[2022-12-16T22:39:35.675+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2022-12-16T22:38:47.807196+00:00 [queued]>
[2022-12-16T22:39:35.676+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-16T22:39:35.677+0000] {taskinstance.py:1284} INFO - Starting attempt 2 of 3
[2022-12-16T22:39:35.678+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-16T22:39:35.754+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): create_dashboard_task> on 2022-12-16 22:38:47.807196+00:00
[2022-12-16T22:39:35.773+0000] {standard_task_runner.py:55} INFO - Started process 8825 to run task
[2022-12-16T22:39:35.780+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_etl_pipeline', 'create_dashboard_task', 'manual__2022-12-16T22:38:47.807196+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/titanic_etl.py', '--cfg-path', '/tmp/tmpowlwbngm']
[2022-12-16T22:39:35.784+0000] {standard_task_runner.py:83} INFO - Job 68: Subtask create_dashboard_task
[2022-12-16T22:39:35.939+0000] {task_command.py:389} INFO - Running <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2022-12-16T22:38:47.807196+00:00 [running]> on host 20ab0680b7da
[2022-12-16T22:39:36.124+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_etl_pipeline
AIRFLOW_CTX_TASK_ID=create_dashboard_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-16T22:38:47.807196+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-16T22:38:47.807196+00:00
[2022-12-16T22:39:36.196+0000] {logging_mixin.py:137} INFO - Dash is running on http://127.0.0.1:8050/
[2022-12-16T22:39:36.195+0000] {dash.py:1968} INFO - Dash is running on http://127.0.0.1:8050/

[2022-12-16T22:39:36.217+0000] {logging_mixin.py:137} INFO -  * Serving Flask app '***'
[2022-12-16T22:39:36.218+0000] {logging_mixin.py:137} INFO -  * Debug mode: on
[2022-12-16T22:39:36.242+0000] {logging_mixin.py:137} WARNING - Address already in use
[2022-12-16T22:39:36.243+0000] {logging_mixin.py:137} WARNING - Port 8050 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2022-12-16T22:39:36.272+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-16T22:39:36.322+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
